.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: 
+      - python
+env:
+      - CC_TEST_REPORTER_ID=873429bec9fd50de3f9c12a9ff621f5a560d04d2ef28e9f6d15bd00c9ba598d5
+
+python:
+      - 3.6
+
+virtualenv:
+      system-site-packages: false
+
+# Install dependencies
+install:
+      - pip3 install -r requirements.txt
+      - pip3 install coveralls
+
+script: 
+      
+
+before_script:
+      - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
Is this a necessary command for the test coverage to work?

Reply:
Yeah. That is for code climate test coverage